src/components/color_picker/color_picker.xml

```diff
@@ -1,80 +1,78 @@
 <templates>
   <t t-name="o-spreadsheet-ColorPicker" owl="1">
-    <div
-      class="o-color-picker"
-      t-att-class="props.dropdownDirection || 'right'"
-      t-on-click.stop=""
-      t-att-style="colorPickerStyle">
-      <div class="o-color-picker-section-name">Standard</div>
-      <div class="colors-grid">
-        <div
-          t-foreach="COLORS"
-          t-as="color"
-          t-key="color"
-          class="o-color-picker-line-item"
-          t-att-data-color="color"
-          t-on-click="() => this.onColorClick(color)"
-          t-attf-style="background-color:{{color}};">
+    <Popover t-props="popoverProps">
```

Comment:
you only wrapped it in a popover tag right ?

Reply:
That and removed the `t-att-class="props.dropdownDirection || 'right'"`. It's a bit annoying that the diff becomes unreadable after adding indentation :x

Reply:
Today I learned that you can do a `git diff --ignore-all-space`. It's also available in github.com
![image](https://user-images.githubusercontent.com/31045223/222764329-e569eab2-f948-4fd2-bc08-c7818d6f57b5.png)

Reply:
ouh, clean, thanks !